src/standalone/Bucket/Bucket.jl

```diff
@@ -389,8 +402,8 @@ function set_initial_parameter_field!(
     p,
     surface_coords,
 ) where {FT}
-    (; α_sfc) = albedo
-    @. p.bucket.α_sfc = α_sfc(surface_coords)
+    # TODO should we include snow here?
```

Comment:
I think yes, for the Function and the Static cases. the surface albedo in those cases is the linear combination of bareground & snow albedo

Reply:
wait - can we get rid of the `set_initial_parameter_field` altogether? In update aux we will compute the linear combination and update p.bucket.alpha_surface for all models.

Reply:
I think we can for the Static and Function cases, because we have already set the bareground parameter field in the constructor. but we have to see if `next_albedo` does the right thing in this case: https://github.com/CliMA/ClimaLand.jl/blob/f05f861a8eed75d45cc0686333333b912137260b/src/standalone/Bucket/Bucket.jl#L452

Reply:
I think we can! I changed the logic for next_albedo to handle the initialization case and verified locally that it produces the same field as the current `set_initial_parameter_field!`